docs/typelibrary/utils/sequence-control/Schieber/SchieberVerriegelungComposite.md

```
@@ -56,4 +58,4 @@ Im Gegensatz zu einfachen Verriegelungs-FBs bietet dieser Composite-Baustein:
 - Kapselung der gesamten Verriegelungslogik
 
 ## Fazit
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)